forLim.py

```diff
@@ -448,15 +442,35 @@ def check_polylines_path(self):
                 self.messageBar.pushMessage("Fichier polylignes", "Le fichier specifie n'existe pas.", QgsMessageBar.CRITICAL, 7)
         else:
             self.dlg.CBox_polylinesBuffer.clear()
-    
-    
+
+### Method Choice
+    #Choose Convolution window (FC) or Delaunay's Triangulation (TR)
+    def select_tr_method(self):
```

Comment:
Well, that's a detail, but as you check the radio button state anyway, you could group that in a single function